### PR TITLE
Add automated test for TC-ACE-1.3 in python

### PR DESCRIPTION
**Pull request by cecille**
This test requires python as it requires the allocation of three controllers on one fabric with CAT tags. This cannot be specified in the yaml.

### CODE REVIEW

**Comment by cecille** [comment]
Note - matches to test plan PR https://github.com/CHIP-Specifications/chip-test-plans/pull/2363

File: src/python_testing/TC_ACE_1_3.py [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+        logging.info('cat1v1 0x%x', cat1v1);
+
+        self.print_step(1, "Commissioning, already done")
+        TH0 = self.default_controller
+        fabric_admin = self.certificate_authority_manager.activeCaList[0].adminList[0]
+
+        TH0_nodeid = self.matter_test_config.controller_node_id
+        TH1_nodeid = self.matter_test_config.controller_node_id + 1
+        TH2_nodeid = self.matter_test_config.controller_node_id + 2
+        TH3_nodeid = self.matter_test_config.controller_node_id + 3
```
[the remark this answers is not in the document]
Comment (cecille) [reply src/python_testing/TC_ACE_1_3.py:68]: See https://github.com/CHIP-Specifications/chip-test-plans/pull/2363 - if we do it this way, we don't need any CATs on the commissioner and we can use the default commissioning process.

File: src/python_testing/TC_ACE_1_3.py [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+                                         paaTrustStorePath=str(self.matter_test_config.paa_trust_store_path),
+                                         catTags=[cat1v3])
+        TH2 = fabric_admin.NewController(nodeId=TH2_nodeid,
+                                         paaTrustStorePath=str(self.matter_test_config.paa_trust_store_path),
+                                         catTags=[cat1v2, cat2v1])
+        TH3 = fabric_admin.NewController(nodeId=TH3_nodeid,
+                                         paaTrustStorePath=str(self.matter_test_config.paa_trust_store_path),
+                                         catTags=[cat1v1, cat2v2])
+
+        self.print_step(2, "TH0 write sACL all view on EP1")
```
[the remark this answers is not in the document]
Comment (cecille) [reply src/python_testing/TC_ACE_1_3.py:80]: This is following the current test plan. We can change it to a PIXIT, but I'd like to confirm whether a device can skip endpoint IDs. The cluster used here is required on every EP

[the remark this answers is not in the document]
Comment (cecille) [reply src/python_testing/TC_ACE_1_3.py]: Test plan changed to feature EP0 only.

File: src/python_testing/TC_ACE_1_3.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mlepage-google) [reply src/python_testing/TC_ACE_1_3.py:66]: Yes. The purpose of using 3 nodes is to test that you can specify more than one (i.e., 2) in a list of subjects, and still have one more to test that it doesn't have privilege. Any of those 3 nodes could be the admin/commissioner (and still manipulate the ACLs to remove its own privileges for the cluster in the tests) but if it's easier to use a 4th node as admin/commissioner, due to the way CATs are installed in credentials, that's fine and appropriate.

File: src/python_testing/matter_testing_support.py [hunk 3]
```diff
@@ -268,6 +270,37 @@ async def read_single_attribute(self, dev_ctrl: ChipDeviceCtrl, node_id: int, en
...
+        result = await dev_ctrl.ReadAttribute(node_id, [(endpoint, attribute)])
+        attr_ret = result[endpoint][cluster][attribute]
+        err_msg = "Did not see expected error when reading {}:{}".format(str(cluster), str(attribute))
+        asserts.assert_true(attr_ret is not None, err_msg)
+        asserts.assert_true(isinstance(attr_ret, Clusters.Attribute.ValueDecodeFailure), err_msg)
+        asserts.assert_true(isinstance(attr_ret.Reason, InteractionModelError), err_msg)
+        asserts.assert_equal(attr_ret.Reason.status, error, err_msg)
+        return attr_ret
+
+    def print_step(self, stepnum: int, title:str) -> None:
```
Comment (mlepage-google) [inline src/python_testing/matter_testing_support.py:301]: Spacing, looks odd to have both styles in same line.

**Review by mlepage-google — APPROVED** [review]
Matches the (updated) test plan, thanks Cecille!

File: src/python_testing/matter_testing_support.py (hunk 3 above)
Comment (cecille) [reply src/python_testing/matter_testing_support.py]: Turns out my auto-formatter was off. New commit fixes this.

**Review by yufengwangca — APPROVED** [review]

**Pull request merged**